### PR TITLE
ADD:cran-checkpoint

### PR DESCRIPTION
**Pull request by gwaygenomics**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

Adding CRAN checkpoint package

### CODE REVIEW

**Comment by gwaygenomics** [comment]
@bgruening @johanneskoester looks like this is good to merge! Thanks!

File: recipes/r-checkpoint/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,96 @@
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
```
Comment (daler) [inline recipes/r-checkpoint/meta.yaml]: can you remove these windows-specific lines please?

File: recipes/r-checkpoint/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    - r-base
+
+  run:
+    - r-base
+
+test:
+  commands:
+    # You can put additional test commands to be run here.
+    - $R -e "library('checkpoint')" # [not win]
+    - "\"%R%\" -e \"library('checkpoint')\"" # [win]
```
Comment (daler) [inline recipes/r-checkpoint/meta.yaml]: more windows-specific lines

File: recipes/r-checkpoint/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+# VignetteBuilder: knitr
+# RoxygenNote: 6.0.1
+# NeedsCompilation: no
+# Packaged: 2017-04-12 12:20:17 UTC; adevries
+# Repository: CRAN
+# Date/Publication: 2017-04-12 13:32:01 UTC
+
+# See
+# http://docs.continuum.io/conda/build.html for
+# more information about meta.yaml
```
Comment (daler) [inline recipes/r-checkpoint/meta.yaml]: can you also remove extra comments? https://bioconda.github.io/guidelines.html#comments-in-meta

**Comment by daler** [comment]
Also, I didn't know about this package. Looks very useful in general, and nice to know Microsoft is snapshotting CRAN

**Comment by gwaygenomics** [comment]
> Also, I didn't know about this package. Looks very useful in general, and nice to know Microsoft is snapshotting CRAN

Yes, it has been useful in the past. Although MRAN is snapshots by date; which is not ideal for specific package versioning. By the same token, its handy for reproducibility since you only need to provide other users with a date and the package will do the rest.

Unfortunately, there is no way to version `checkpoint` hence the addition to bioconda!

**Comment by gwaygenomics** [comment]
Thanks @daler, I removed the comments and it looks like it passed CI again

**Comment by daler** [comment]
looks good, thanks!

**Pull request merged**